src/server/game/Scripting/ScriptMgr.h

```diff
@@ -1171,6 +1171,55 @@ class TC_GAME_API ScriptMgr
         std::string _currentContext;
 };
 
+template <class _SpellScript>
+struct SpellScriptLoaderEx : public SpellScriptLoader
+{
+    SpellScriptLoaderEx(const char* name) : SpellScriptLoader(name) {}
```

Comment:
codestyle: const after type and one space between {}
`SpellScriptLoaderEx(char const* name) : SpellScriptLoader(name) { }`